src/main/groovy/wooga/gradle/node/NodeReleasePlugin.groovy

```diff
@@ -109,4 +130,22 @@ class NodeReleasePlugin implements Plugin<Project> {
         task.config = [version: project.getVersion().toString()]
         task.description = "Set 'package.json' version based on release plugin version"
     }
+
+    private void configureNpmCredentialsTasks(Project project, NodeReleasePluginExtension extension) {
+        project.tasks.withType(NpmCredentialsTask, new Action<NpmCredentialsTask>() {
+
+            @Override
+            void execute(NpmCredentialsTask npmCredentialsTask) {
+                configureNpmCredentialsTask(project, extension, npmCredentialsTask)
+            }
+        })
+    }
+
+    private void configureNpmCredentialsTask(Project project, NodeReleasePluginExtension extension, NpmCredentialsTask npmCredentialsTask) {
```

Comment:
Quite excessive use of methods here.
Why not add the body in the `execute` methods?